client/my-sites/checkout/concierge-quickstart-session/index.jsx

```diff
@@ -0,0 +1,397 @@
+								<div className="concierge-quickstart-session__placeholder-row is-placeholder" />
+							</div>
+						</div>
+					</CompactCard>
+				) : (
+					''
+				) }
+				<CompactCard>
+					<div className="concierge-quickstart-session__placeholders">
+						<>
+							<div className="concierge-quickstart-session__placeholder-row is-placeholder" />
+							<div className="concierge-quickstart-session__placeholder-row is-placeholder" />
+							<div className="concierge-quickstart-session__placeholder-row is-placeholder" />
+							<div className="concierge-quickstart-session__placeholder-row is-placeholder" />
+						</>
+					</div>
+				</CompactCard>
+				<CompactCard>
+					<div className="concierge-quickstart-session__footer">
+						<div className="concierge-quickstart-session__placeholders">
+							<div className="concierge-quickstart-session__placeholder-button-container">
+								<div className="concierge-quickstart-session__placeholder-button is-placeholder" />
+								<div className="concierge-quickstart-session__placeholder-button is-placeholder" />
+							</div>
+						</div>
+					</div>
+				</CompactCard>
+			</>
+		);
+	}
+
+	header() {
+		const { translate } = this.props;
+		return (
+			<header className="concierge-quickstart-session__header">
+				<h2 className="concierge-quickstart-session__title">
+					{ translate( 'Hold tight, your site is being upgraded…' ) }
+				</h2>
+			</header>
+		);
+	}
+
+	body() {
+		const { translate, productCost, productDisplayCost, currencyCode } = this.props;
+		const fullCost = Math.round( productCost * 2 );
+		//const savings = fullCost - productCost;
+		return (
+			<Fragment>
+				<h4 className="concierge-quickstart-session__sub-header">
+					{ translate( 'Presenting… a personal WordPress Expert, by your side' ) }
+				</h4>
+				<div className="concierge-quickstart-session__column-pane">
+					<div className="concierge-quickstart-session__column-content">
+						<p>
+							{ translate(
+								"What if you could sit with a true expert, someone who's helped hundreds of people succeed online, and get their help to build a great site… in less time you ever thought possible?"
+							) }
+						</p>
+						<p>
+							<b>
+								{ translate( 'Introducing WordPress.com {{i}}Quick Start{{/i}} Sessions.', {
+									components: { i: <em /> },
+								} ) }
+							</b>
+						</p>
+						<p>
+							{ translate(
+								"{{i}}Quick Start{{/i}} sessions are 45-minute one-on-one conversations between you and one of our website building experts. They know WordPress inside out and will help you achieve your goals with a smile. That's why we call them Happiness Engineers.",
+								{
+									components: { i: <em /> },
+								}
+							) }
+						</p>
+						<p>
+							{ translate(
+								'On these sessions, a Happiness Engineer will show you everything you need to build a great site in the fastest way possible, including:'
```

Comment:
This copy kind of promises the session will go over all of these things. Even though they're really more customized to what the user wants.
Do you think the HEs doing these session will need to know that the user saw this upsell page vs the control?

Reply:
Also, feel free to ignore any comments I leave on the copy. Just my 2¢.

Reply:
Also good points. 

The assumption here is that is too early for people to see the value of the session (i.e. they haven't faced any real issues yet) that's why the copy kind of leans into painting a more "guided" picture. I've touched a little bit the copy to balance this. 

Also, yes, HEs will get up to speed to know this test is running.